state/migration_export.go

```diff
@@ -1006,7 +1017,7 @@ func (e *exporter) relations() error {
 					continue
 				}
 				key := ru.key()
-				if !relationScopes.Contains(key) {
+				if !e.cfg.SkipRelationScope && !relationScopes.Contains(key) {
```

Comment:
If we are skipping relation scopes, we don't need to do

relationScopes, err := e.readAllRelationScopes()

above. I think the bool should really  be skipRelationScopeCheck? Since we aren't exporting them but just checking